kuma/users/models.py

```diff
@@ -215,6 +216,12 @@ def create_user_profile(sender, instance, created, **kwargs):
         p, created = UserProfile.objects.get_or_create(user=instance)
 
 
+@receiver(models.signals.post_save, sender=settings.AUTH_USER_MODEL)
+def invalidate_gravatar_url(sender, instance, created, **kwargs):
+    if instance.email:
+        UserGravatarURLJob().invalidate(instance.email)
```

Comment:
Is it possible for the email to go from having a real value to `None` and we should invalidate that also?